examples/cors/backend/docker-compose.yaml

```diff
@@ -7,6 +7,8 @@ services:
       dockerfile: Dockerfile-frontenvoy
     volumes:
       - ./front-envoy.yaml:/etc/front-envoy.yaml
+    environment:
+      ENVOY_UID: 0    
```

Comment:
Can we change the access log path to `/dev/stdout` instead?

```yaml
          access_log:
            - name: envoy.access_loggers.file
              typed_config:
                "@type": type.googleapis.com/envoy.config.accesslog.v2.FileAccessLog
                path: "/dev/stdout"
```

Reply:
I will do that but If the user will modify path other than "/dev/stdout" then the issue will be the same.

Reply:
yep, i think we should add some docs re log permissions for that reason